[2025-02-06T15:41:01.376+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T15:41:01.387+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T15:41:00.352353+00:00 [queued]>
[2025-02-06T15:41:01.393+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T15:41:00.352353+00:00 [queued]>
[2025-02-06T15:41:01.399+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T15:41:01.413+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-06 15:41:00.352353+00:00
[2025-02-06T15:41:01.418+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8120) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T15:41:01.420+0000] {standard_task_runner.py:72} INFO - Started process 8122 to run task
[2025-02-06T15:41:01.420+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'extract_data', 'manual__2025-02-06T15:41:00.352353+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmp7reiwj76']
[2025-02-06T15:41:01.422+0000] {standard_task_runner.py:105} INFO - Job 114: Subtask extract_data
[2025-02-06T15:41:01.455+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.extract_data manual__2025-02-06T15:41:00.352353+00:00 [running]> on host b176106a6d6f
[2025-02-06T15:41:01.613+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-06T15:41:00.352353+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-06T15:41:00.352353+00:00'
[2025-02-06T15:41:01.614+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T15:41:02.091+0000] {dag_que_faire_a_paris.py:40} INFO - âœ… Extraction completed successfully!
[2025-02-06T15:41:02.093+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-06T15:41:02.098+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T15:41:02.099+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=extract_data, run_id=manual__2025-02-06T15:41:00.352353+00:00, execution_date=20250206T154100, start_date=20250206T154101, end_date=20250206T154102
[2025-02-06T15:41:02.117+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T15:41:02.136+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T15:41:02.140+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
